scripts/scil_compute_mean_fixel_lobe_metric_from_bundles.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+Given a bundle and Bingham coefficients, compute the average lobe-specific
+metric at each voxel intersected by the bundle.
```

Comment:
Can you add a sentence or 2 to describe how the intersection between bundle segment in the voxel and lobe orientation is done?